web/static/humans.txt

```diff
@@ -51,6 +51,7 @@ Supabase would not be possible without the work of some other amazing open sourc
 - PostgREST - https://github.com/PostgREST
 - Kong - https://github.com/kong/
 - Next.js - https://nextjs.org/
+- Tailwindcss - https://tailwindcss.com/
```

Comment:
Seems like the brand name is stylised as `Tailwind CSS`:

![image](https://user-images.githubusercontent.com/3143132/149773417-55a0d816-7a7a-4c2f-9750-99286125b071.png)

```suggestion
- Tailwind CSS - https://tailwindcss.com/
```